packages/pages/src/vite-plugin/plugin.ts

```diff
@@ -22,6 +27,7 @@ const plugin = async (): Promise<PluginOption[]> => {
         process: "build",
       },
     }),
+    ...(hasPublicAssets ? [copyPublicAssets(projectStructure)] : []),
```

Comment:
Does this need to return an array that you spread? Can you do this?
`hasPublicAssets ? copyPublicAssets(projectStructure) : null` or maybe `""` instead of null